docs/ossec_alerts.rst

```diff
@@ -527,18 +551,43 @@ securedrop@freedom.press for help.
 
 .. _SecureDrop Support Portal: https://securedrop-support.readthedocs.io/en/latest/
 
-Uncommon OSSEC Alerts
+Daily reports
+^^^^^^^^^^^^^
+
+On days where file integrity checksums have changed or users have logged into ``app``
+or ``mon`` servers, you will receive emails entitled ``Daily report: File changes`` or
+``Daily report: Successful logins``. These emails may be a more digestible format
+should you not have continuous access to the inbox or GPG key.
+
+**Action**: periodically review these daily reports to ensure file changes correspond
+to platform updates and logins correspond to authorized admin activity on the SecureDrop
+servers.
+
+If you have any suggestions on how to further tune or improve the alerting,
+you can join the discussion or open an issue on `GitHub <https://github.com/freedomofpress/securedrop/labels/goals%3A%20reduce%20IDS%20noise>`__.
```

Comment:
Where would they join the discussion? We could link to the forum, but I think it might be simplest to just direct them to open an issue.